PhotoGether/PresentationLayer/EditPhotoRoomFeature/EditPhotoRoomFeature/Source/StickerBottomSheetViewController.swift

```diff
@@ -0,0 +1,127 @@
+import Combine
+import UIKit
+
+import BaseFeature
+import DesignSystem
+import PhotoGetherDomainInterface
+
+protocol StickerBottomSheetViewControllerDelegate {
+    func stickerBottomSheetViewController(_ viewController: StickerBottomSheetViewController, didTap emoji: EmojiEntity)
+}
+
+final class StickerBottomSheetViewController: UIViewController, ViewControllerConfigure {
+    private let collectionView: StickerCollectionView
+    private let viewModel: StickerBottomSheetViewModel
+    
+    var delegate: StickerBottomSheetViewControllerDelegate?
```

Comment:
```suggestion
    weak var delegate: StickerBottomSheetViewControllerDelegate?
```
weak가 없으면 순환참조가 생길 수 있어요

Reply:
[5ed57df](https://github.com/boostcampwm-2024/iOS04-PhotoGether/pull/98/commits/5ed57df0a31365734bca6da5706dd114fd502d6d)

그렇네요 🥲

수정 완료했습니다!